web/src/engine/FetchProviderNodeWebkit.ts

```diff
@@ -46,6 +46,30 @@ function ModifyRequestHeaders(details: chrome.webRequest.WebRequestHeadersDetail
 }
 
 function ModifyResponseHeaders(details: chrome.webRequest.WebResponseHeadersDetails): chrome.webRequest.BlockingResponse {
+
+    //Handle Set-Cookie header according to https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Set-Cookie
+    const cookiesHeaders = details.responseHeaders.filter(header => header.name.toLowerCase() == 'set-cookie');
+
+    //Multiple Set-Cookie headers can be sent in the same response.
+    if (cookiesHeaders.length > 0) {
```

Comment:
no need for this `if` as  `foreach` will iterate only on elements.
filter can be `null` then just use `cookiesHeaders?.forEach`